### PR TITLE
[#176116102] Add io-functions-paymentmanager

### PR DESCRIPTION
**Pull request by balanza**

### CODE REVIEW

File: io-functions-paymentmanager/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM functions
+
+ARG FUNCTIONS_PUBLIC_COMMIT_SHA
```
Comment (BurnedMarshal) [inline io-functions-paymentmanager/Dockerfile:3]:
```suggestion
ARG FUNCTIONS_PAYMENTMANAGER_COMMIT_SHA
```

**Review by BurnedMarshal — CHANGES_REQUESTED** [review]
Please add the repo configuration inside `config/repositories.json` file

File: docker-compose.yml [hunk 2]
```diff
@@ -187,6 +187,28 @@ services:
...
 
+  functions-paymentmanager:
+    image: io-functions-paymentmanager
+    build:
+      context: ${FUNCTIONS_PAYMENTMANAGER_CONTEXT}
+      dockerfile: ./Dockerfile
+      args:
+        - FUNCTIONS_PAYMENTMANAGER_COMMIT_SHA
+    env_file:
+      - generated-env/env.cosmosdb
```
Comment (balanza) [inline docker-compose.yml:198]: @BurnedMarshal do we need this?

Comment (BurnedMarshal) [reply docker-compose.yml:198]: Yes, this function use `ProfileModel`

**Pull request merged**